Fabulous.XamarinForms/src/Fabulous.XamarinForms/Xamarin.Forms.Core.json

```diff
@@ -2316,6 +2320,59 @@
           "inputType": "obj * obj * Xamarin.Forms.ScrollToPosition * Fabulous.XamarinForms.AnimationKind",
           "modelType": "obj * obj * Xamarin.Forms.ScrollToPosition * Fabulous.XamarinForms.AnimationKind",
           "updateCode": "(fun _ curr target -> triggerScrollTo curr target)"
+        },
+        {
+          "name": "Footer",
+          "defaultValue": "null"
+        },
+        {
+          "name": "FooterTemplate",
```

Comment:
Could you declare only the direct property and not the Template one (here `Footer` vs `FooterTemplate`, and the others as well) please ?

We don't provide direct support for Template properties (DataTemplate, etc.).

Instead we only expose one property that accepts a ViewElement that we put directly in the "real" property (e.g. Footer) since we don't need to use BindingContext.

Reply:
So I only have to delete those parts, correct?

Reply:
Yes